src/flask_container_scaffold/app_scaffold.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from flask import Flask
+
+from flask_container_scaffold.util import load_yaml
+
+
+class AppScaffold(object):
+
+    def __init__(self, app=None,
+                 name=__name__, config=None):
+        Flask.jinja_options = dict(Flask.jinja_options, trim_blocks=True,
+                                   lstrip_blocks=True)
+        self.app = app or Flask(name, instance_relative_config=True)
+        self.config = config
+        self._init_app(app)
+
+    def _init_app(self, app):
+        self._setup_instance_folder()
+        self._configure_app()
+
+    # NOTE: This method is old, and likely to go away, unless
+    # we make it for dev mode only, as a production app should not
+    # need to create folders.
+    def _setup_instance_folder(self):
+        # Ensure the instance folder exists
+        try:
+            os.makedirs(self.app.instance_path)
+        except OSError:
+            pass
+
+    def _configure_app(self):
+        if self.config is None:
+            # load the instance config, if it exists, when not testing
+            self.app.config.from_pyfile('settings.cfg', silent=True)
+        else:
+            # load the test config if passed in
+            self.app.config.from_mapping(self.config)
+            # Next, load from override file, if specified
+        if os.environ.get('SETTINGS'):
```

Comment:
Might be worth adding a note in the README about the `SETTINGS` env variable and it's usage.